application/models/user.go

```diff
@@ -428,6 +428,28 @@ func (u *User) AttachPhoto(fileID string) (File, error) {
 	return f, nil
 }
 
+// RemovePhoto removes an attached photo from the User profile
+func (u *User) RemovePhoto() error {
+	if u.ID < 1 {
+		return fmt.Errorf("invalid User ID %d", u.ID)
+	}
+
+	oldID := u.PhotoFileID
```

Comment:
Would it be better to have, 
```
if !oldID.Valid {
     return nil
}
```
and then do the UpdateColumns and unlink the file.

Reply:
Probably. As you've probably noticed, I naturally think this way. Maybe a holdover from old C-language tradition?